renku/domain_model/workflow/plan.py

```diff
@@ -135,6 +136,8 @@ def is_derivation(self) -> bool:
 class Plan(AbstractPlan):
     """Represent a `renku run` execution template."""
 
+    annotations: List[Annotation] = list()
```

Comment:
Is this needed?

Reply:
yes, this is for backwards compatibility. Plans created before this change will automatically have `annotations` set to an empty list when loaded from the database